docs/services/navigation/_index.md

```diff
@@ -554,3 +554,124 @@ obstacles = await my_nav.get_obstacles()
 
 {{% /tab %}}
 {{< /tabs >}}
+
+## Concepts
+
+The following concepts are important to understand when utilizing the navigation service.
+Each concept is a type of relative or absolute measurement, taken by a [movement sensor](/components/movement-sensor/), which can then be utilized by your robot to navigate across space.
+
+Here's how to make use of the following types of measurements:
+
+### Compass Heading
+
+Compass heading measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [gps-nmea](/components/movement-sensor/gps/gps-nmea/)
+- [gps-nmea-rtk-pmtk](/components/movement-sensor/gps/gps-nmea-rtk-pmtk/)
+- [gps-nmea-rtk-serial](/components/movement-sensor/gps/gps-nmea-rtk-serial/)
+
+An example of a `Compass Heading` reading:
+
+``` go
+gps.GetCompassHeading{175}
+```
+
+Compass heading readings are taken through the movement sensor API's [`GetCompassHeading()`](/components/movement-sensor/#getcompassheading).
+If you want to read compass headings, [configure a capable movement sensor](/components/movement-sensor/#configuration) on your robot.
+Use compass heading readings to determine the *bearing* of your robot, or, the [cardinal direction](https://en.wikipedia.org/wiki/Cardinal_direction) that your robot is facing.
+
+### Orientation
+
+Orientation measurements are read by the following {{< glossary_tooltip term_id="model" text="models" >}} of [movement sensor](/components/movement-sensor/):
+
+- [imu-wit](/components/movement-sensor/imu/imu-wit/)
+- [imu-vectornav](/components/movement-sensor/imu/imu-vectornav/)
+
+An `Orientation` reading specifies the orientation of an object in 3D space as an "orientation vector", or, its position within the [cartesian coordinate system](https://en.wikipedia.org/wiki/Cartesian_coordinate_system) relative to some specific `origin` point that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
```

Comment:
```suggestion
An `Orientation` reading specifies the orientation of an object in 3D space as an *orientation vector* relative to some specific `origin` orientation that you, the user, need to choose and configure for your robot in Viam's [frame system](/services/frame-system/).
```
Bringing position into this seems misleading. As far as the use of "point," can we get an SME to review language? Orientation isn't defined by or relative to a point since a point is more about position, but not sure my suggestion is ideal either.

Reply:
I just removed a lot of the wording here, ty! done

Reply:
Hmm it looks the sameish. Making a suggestion.